dev-python/pypubsub/pypubsub-4.0.3.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{4,5,6,7} )
```

Comment:
Don't add support for python-3.4, it is currently masked and getting removed in ~2 weeks.